app/javascript/hooks/mutations/room_settings/useUpdateRoomSetting.jsx

```diff
@@ -16,12 +16,23 @@ export default function useUpdateRoomSetting(friendlyId) {
     return axios.patch(`/room_settings/${friendlyId}.json`, data);
   };
 
+  // Returns a more suiting toast message if the updated room setting is an access code
+  const toastSuccess = (variables) => {
+    if ((variables.settingName === 'glModeratorAccessCode' || variables.settingName === 'glViewerAccessCode') && variables.settingValue === true) {
+      return toast.success(t('toast.success.room.access_code_generated'));
+    }
+    if ((variables.settingName === 'glModeratorAccessCode' || variables.settingName === 'glViewerAccessCode') && variables.settingValue === false) {
+      return toast.success(t('toast.success.room.access_code_deleted'));
+    }
+    return toast.success(t('toast.success.room.room_setting_updated'));
+  };
```

Comment:
This logic could be used in a couple of place to improve the Toast relevance - mostly in SiteSettings and RoomSettings.

As of now, if you update any SiteSetting, a toast will pop up with: "The site setting has been updated".

But, it could instead be: "The branding image has been deleted." if you remove the branding image for example.

It improves the Toast/UX but it adds some code complexity.